src/http/index.js

```diff
@@ -26,17 +27,28 @@ class HttpCommunication {
       this.contextStorage = contextStorage;
     }
 
+    static getRequestContext(req, customContextValue) {
+      return {
+        traceId: (req.headers && req.headers['X-Q-TRACEID']) ? req.headers['X-Q-TRACEID'] : uuid(),
+        userId: (req.user && req.user.userId) ? String(req.user.userId) : null,
+        ab: (req.headers && req.headers['Q-AB-ROUTE']) ? req.headers['Q-AB-ROUTE'] : null,
+        reqStartTime: process.hrtime(),
```

Comment:
its being deprecated https://nodejs.org/docs/latest-v16.x/api/process.html#processhrtimebigint

Reply:
Do you think we should use `hrtime.bigint();`